src/Workleap.OpenApi.MSBuild/ValidateOpenApiTask.cs

```diff
@@ -75,6 +87,27 @@ protected override async Task<bool> ExecuteAsync(CancellationToken cancellationT
         return true;
     }
 
+    private bool CheckIfBaseSpecExists()
+    {
+        foreach (var file in this.OpenApiSpecificationFiles)
+        {
+            if (File.Exists(file))
+            {
+                continue;
+            }
+
+            this.Log.LogWarning(
```

Comment:
Should we copy it for them? Initialize their current spec file! We could write a comment "we have initialize your spec file, it will now be used as the base comparaison"

Reply:
I think it could be a better UX

Reply:
I don't think we should. It is a one-time task so not every dev will have to do go through this. Plus I think developers would want freedom of where their specs are saved. In which case they will have to "copy + paste" anyway. Maybe we can add this for future enhancement or prioritize based on user-feedback